recipe/meta.yaml

```
@@ -1,43 +1,55 @@
 {% set name = "weave" %}
-{% set version = "0.17.0" %}
-{% set sha256 = "2703f3ae6d23ad47b5f09b6bcb7affd0fb587120a0c973e7be40ef24de709998" %}
+{% set version = "0.51.31" %}
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - python_requires.patch
+  sha256: 66df71c5865a631986c346d24c7f9b1d22126bd266ad59d1610f26aa7487b025
 
 build:
-  number: 4
-  skip: True  # [py3k or not x86_64]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
+  skip: true  # [py<39]
+  # we don't have 'wandb' for s390x
+  skip: true  # [s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
  entry_points:
    - weave = weave.trace.cli:cli
```
see https://github.com/wandb/weave/blob/v0.51.24/pyproject.toml#L111-L112, and because we run in tests `weave --help`